oauth-implicit/go/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Implicit Login</title>
+        <script src="https://sdk-cdn.mypurecloud.com/external/go-cli/axios/0.23.0/axios.min.js"></script>
```

Comment:
We should use a public CDN here. We don't really intend for people to consume axios from our CDN directly in their custom apps, and we don't provide any documentation around our rehosting of it.